qiskit_experiments/framework/base_experiment.py

```diff
@@ -23,6 +23,7 @@
 from qiskit.exceptions import QiskitError
 from qiskit.qobj.utils import MeasLevel
 from qiskit.providers.options import Options
+from qiskit_ibm_runtime import SamplerV2 as Sampler
```

Comment:
Previously, we had carefully kept `qiskit_ibm_runtime` from being a required dependency. It needs to be added to `requirements.txt` if it is being imported top-level like this.